proxy/manager_test.go

```diff
@@ -62,3 +66,57 @@ func TestProxyHandleRequestError(t *testing.T) {
 		})
 	})
 }
+
+func TestIsReleaseCalendarURl(t *testing.T) {
+	Convey("Given a list of URLs", t, func() {
+		releaseCalendarEndpoints := []TestCase{
+			{
+				uri:      "/releases/",
+				expected: true,
+			},
+			{
+				uri:      "/releases/adoption",
+				expected: true,
+			},
+			{
+				uri:      "/releasecalendar",
```

Comment:
Read the ticket again, please, because this is not correct. It's only `/releases/xyz` that we need to forward.